Dominus/Engine/GLFWInputHandler.cpp

```diff
@@ -0,0 +1,135 @@
+//
+//  GLFWInputHandler.cpp
+//  Dominus
+//
+//  Created by Alvaro Chambi Campos on 20/2/16.
+//  Copyright © 2016 frikazos. All rights reserved.
+//
+
+#include "GLFWInputHandler.h"
+#include "Log.hpp"
+
+GLFWInputHandler* instance;
+
+GLFWInputHandler::GLFWInputHandler( GLFWwindow* window ) :
+                                                        window( window ),
+                                                        onDrag( false ),
+                                                        onHoldKey( false ){
+    instance = this;
+    event = new Event;
+    xSaved = 0;
+    ySaved = 0;
+}
+
+GLFWInputHandler::~GLFWInputHandler() {
+
```

Comment:
Should we release Window or other members here? We should start using unique_ptr or share_ptr to avoid problems.